lametro/templates/lametro/index.html

```diff
@@ -86,6 +86,29 @@ <h2>
                 </a>
             </div>
         </div>
+        <div class="row-fluid">
+            <div class='col-sm-10 col-sm-offset-1'>
+                <h2>
+                    <span class="non-mobile-only"><i class="fa fa-calendar-check-o" aria-hidden="true"></i>
+</span>
+                    Recent Meetings
+                </h2>
+                {% if most_recent_past_meetings %}
+                    {% if most_recent_past_meetings %}
```

Comment:
why is this conditional duplicated?

Reply:
Ditto this question.

Reply:
It shouldn't be. Pretty sure it's just a mistake on my end 😅